src/index.tsx

```diff
@@ -61,7 +63,15 @@ addMenuActionSet(ContextMenuKind.TRASH, trashActionSet);
 fetchConfig()
     .then(({ config, apiHost }) => {
         const history = createBrowserHistory();
-        const services = createServices(config);
+        const services = createServices(config, {
+            progressFn: (id, working) => {
+                store.dispatch(progressIndicatorActions.TOGGLE_WORKING({ id, working }));
+            },
+            errorFn: (id, error) => {
+                console.error("Backend error:", error);
+                store.dispatch(snackbarActions.OPEN_SNACKBAR({ message: "Backend error", kind: SnackbarKind.ERROR }));
```

Comment:
Global errors handler could also be applied directly and only to apiCient so that we would avoid overriding a lot of other classes just to pass ApiActions to them.

Reply:
Agreed, I'll simply comment out dispatching snackbar message here, but I will left the callback, maybe it will be useful some day